Job 15user mentor

Dimitri-Basseguy opened the pull request:
cette fois sa passe ?

Dimitri-Basseguy committed:
Update Mentor.php

Dimitri-Basseguy committed:
Update User.php

Dimitri-Basseguy committed:
Update SoftSkill.php

Dimitri-Basseguy committed:
Update StackTech.php

Dimitri-Basseguy committed:
Update api_platform.yaml

Dimitri-Basseguy committed:
Update fixture.yaml

ReCoN-96 reviewed (changes requested):
On a un probléme au niveau de swagger, lorsque l'on va sur localhost/docs  ui swagger ne fonctionne pas et nous avons cela en résultat 
![bugdimitri](https://user-images.githubusercontent.com/48048474/118783745-a69be200-b88f-11eb-8f13-1e7e4c943df0.png)

De plus j'ai remarqué qu'il y a une erreur concernant la serialization dans la console (peut etre est-ce lié) 
![dimitribug2](https://user-images.githubusercontent.com/48048474/118783852-c03d2980-b88f-11eb-8efd-47793df7b279.png)

Dimitri-Basseguy commented:
Surement quand j'ai dit à apiPlatForm de rester en mode Json pure et stopper le JsonLD., Si vous testez les routes API vous recevez bien le Json ?

ReCoN-96 commented:
> Surement quand j'ai dit à apiPlatForm de rester en mode Json pure et stopper le JsonLD., Si vous testez les routes API vous recevez bien le Json ?

Et c'est possible de faire réapparaitre l'ui de swagger car sans ça je ne sais pas comment recupèrer les routes

Dimitri-Basseguy commented:
normalement les routes sont les mêmes (faut relancer les fixtures avant quand même)
tonlocalhost/projet/mentors
tonlocalhost/projet/mentors/{id}

Dimitri-Basseguy commented:
ha merde chez moi ca foire aussi...

Dimitri-Basseguy committed:
suppression migration + nouvelle

proces pour faire marcher cher moi:
- supprimer les migration dans le dossier /migrations
- php bin/console make:migration
- php bin/console doctrine:migrations:migrate
- bin/console hautelook:fixtures:load

la route localhost/docs marche les autre aussi

Dimitri-Basseguy committed:
testing, ok local.
